Bottom: f4ed11363d5ac65fc566d22cabb0ee80c2821c0e
Top:    79c4a3048ff1867ec2bd503639a1b26f1140fafc
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 16:56:22 +0000

draw.h: Remove unused DrawHoles() prototype


---

diff --git a/src/draw.h b/src/draw.h
index e0688ee..bae0cd8 100644
--- a/src/draw.h
+++ b/src/draw.h
@@ -70,7 +70,6 @@ void DrawLayerGroup (int side, const BoxType *drawn_area);
 void DrawPaste (int side, const BoxType *drawn_area);
 void DrawSilk (HID_DRAW *hid_draw, int side, const BoxType *drawn_area);
 void DrawMask (int side, const BoxType *drawn_area);
-void DrawHoles (bool draw_plated, bool draw_unplated, const BoxType *drawn_area);
 void PrintAssembly (int side, const BoxType *drawn_area);
 
 /* TEMPORARY */
